ariadne/cardano/src/Ariadne/Config/CLI.hs

```diff
@@ -196,17 +196,16 @@ getConfig commitHash = do
 
       resolve :: FilePath -> ConfigDirectories -> FilePath -> FilePath
       resolve prefix ConfigDirectories{..} path
-        | isPrefixOf "@DATA" path = replace "@DATA" cdDataDir path
-        | isPrefixOf "@PWD" path = replace "@PWD" cdPWD path
+        | "@DATA" `isPrefixOf` path = replace "@DATA" cdDataDir path
```

Comment:
The check is whether `@DATA` is a prefix, but `replace` modifies *every* occurrence. Is this intended?

Reply:
Yes, we want to replace every occurrence of `@DATA`. We check whether it's a prefix because if it's a prefix we treat this part as absolute, otherwise we treat it as relative.